### DISCUSSION TITLE
Can't save changes in markdown editor.

### DISCUSSION THREAD

**Discussion by AlexTechNoir** [strapi/strapi#7181] [Legacy Topics]
Hello, everyone!

I'm learning Strapi, it's my very first CMS. I'm creating a content for my demo website (blog).

I've created a content type (article) with rich text editor a couple of days ago. Now when I edited the text, hit "save", first it asks to enter a date of publication, despite it was already entered before and after reentering nothing happens. Changes aren't being saving. No signs from system of something else I should do.

Am I missing something?

**Comment by lauriejim** [comment]
Hello! What is the version of Strapi you are using? (and the database)
Do you have a procedure to follow to be able to reproduce the issue on my side?

**Reply by AlexTechNoir** [reply]
Strapi: 3.1.0. SQLite.

To reproduce: create a Strapi account, create a collection type with markdown text, write something, save, then try to edit it. 

After that "Save" button doesn't work.

**Comment by lauriejim** [comment]
Hum it works fine on my side...
What is the content of your text?

**Reply by AlexTechNoir** [reply]
Here's the screenshot:

![0](https://user-images.githubusercontent.com/43852391/88538342-e00d1380-d017-11ea-88b2-6ae047db0198.jpg)

On Slack I've been told that it may be a bug and there is a fix coming soon:

https://github.com/strapi/strapi/pull/7198

**Reply by lauriejim** [reply]
Hum that's strange, everything looks good :/

**Reply by soupette** [reply]
Can you send us the exact steps to reproduce? Does your datetime field as some kind of required validations? because it might come from this as we have identified an issue with this validation.

**Reply by AlexTechNoir** [reply]
After migrating to 3.1.2 ver. everything works properly! Thank you all for your attention!

**Comment by AlexTechNoir** [comment]
I've migrated to 3.1.2 version and now saving works!

If anyone's interested here's the [migration guide](https://strapi.io/documentation/v3.x/migration-guide/migration-guide-3.0.x-to-3.1.x.html)
